src/components/Account/Account.tsx

```diff
@@ -168,6 +168,7 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
             onSubmit: (values) => updateUser({ firstName: values.firstName || '', lastName: values.lastName || '' }),
             content: (section) => (
               <>
+                <h1 className={styles.hideUntilFocus}>{t('nav.account')}</h1>
```

Comment:
Are we sure about this h1? Although 'my account' is a different route, on the screen it is actually a segment of the User screen, with a user menu as well. Should this be an h2 and should we have a User h1?

Reply:
Hmm, but in the user menu this is also called 'Account', so for the user I would say this is more clear. Otherwise we will have a 'User' heading, what information does this convey? I'm not convinced yet that it's clear enough... 🤔

Reply:
Hm, I understand your doubt. Let's go for your approach for now, we could always reconsider!